src/Views/Appearance.vala

```diff
@@ -311,36 +313,39 @@ public class PantheonShell.Appearance : Gtk.Grid {
             var accent_label = new Gtk.Label (_("Accent:"));
             accent_label.halign = Gtk.Align.END;
 
-            var blueberry_button = new ColorButton ("blueberry");
+            var blueberry_button = new PrefersAccentColorButton ("blueberry", pantheon_act, AccentColor.BLUE);
```

Comment:
Instead of passing in the same information twice (the enum and string for accent colors), it would be better to add a `to_string` method to the enum. This way you're only passing in the information once and it always stays in sync